components/patient-data/rest/src/main/java/org/phenotips/data/rest/PatientByLabeledExternalIdentifierResource.java

```diff
@@ -102,7 +102,7 @@ Response updatePatient(String json, @PathParam("label") String label, @PathParam
     @Consumes(MediaType.APPLICATION_JSON)
     @RequiredAccess("edit")
     Response patchPatient(String json, @PathParam("label") String label, @PathParam("id") String id,
-        @QueryParam("create") @DefaultValue("false") String create);
+        @QueryParam("create") @DefaultValue("false") boolean create);
```

Comment:
Technically specifying `@DefaultValue("false")` is not necessary for booleans since its value by default if uninitialized is false, but I think it's good to be explicit about the default value.